website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
 ## Overview
 
 Connecting your GitLab account to dbt Cloud unlocks exciting and compelling functionality in dbt Cloud. Once your GitLab account is connected, you can:
```

Comment:
nit: but this comes off as a lil' cheesy to me
```suggestion
Once your GitLab account is connected to dbt Cloud, you can:
```

Reply:
@dataders  and @schottj how does this sound?
```suggestion
Connecting your GitLab account to dbt Cloud provides convenience and another layer of security to dbt Cloud:
```